### PR TITLE
Ports glass floors from /vg/

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request
vgstation-coders/vgstation13#15299
Ports glass floors from /vg/, slam people on the floor, walk over it with a mech, look into space with it
unknown111

## Why It's Good For The Game

nice floors

## Changelog
:cl: N3X15
add: Added glass floors
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
Can you link the original PR and credit the author please. 
Also, it appears you can deconstruct glass plating with nothing more than a toolbox

**Comment by jupyterkat** [comment]
Well it is a more flimsy floor that can be destroyed by throwing stuff at it or shoot it

**Comment by jupyterkat** [comment]
But you're right, toolbox decon might be iffy

**Review by TheFakeElon — CHANGES_REQUESTED** [review]
Credits

**Comment by jupyterkat** [comment]
@TheFakeElon Already did

**Comment by TheFakeElon** [comment]
well now you have, cheers

**Review by TheFakeElon — APPROVED** [review]

**Comment by TheFakeElon** [comment]
can you make them a bit stronger? maybe add a damage requirement similar to doors and closets so you can't break the floor with easy to find items

**Comment by jupyterkat** [comment]
It's ready now

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 1]
```diff
@@ -123,9 +123,11 @@ GLOBAL_LIST_INIT(pglass_recipes, list ( \
  */
 GLOBAL_LIST_INIT(reinforced_glass_recipes, list ( \
 	new/datum/stack_recipe("windoor frame", /obj/structure/windoor_assembly, 5, time = 0, on_floor = TRUE, window_checks = TRUE), \
-	null, \
 	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/reinforced/unanchored, time = 0, on_floor = TRUE, window_checks = TRUE), \
-	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE) \
+	//austation begin -- glass floors
+	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE), \
```
Comment (TheFakeElon) [inline code/game/objects/items/stacks/sheets/glass.dm:128]: What’s this?

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 2]
```diff
@@ -169,7 +171,10 @@ GLOBAL_LIST_INIT(reinforced_glass_recipes, list ( \
 
 GLOBAL_LIST_INIT(prglass_recipes, list ( \
 	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/plasma/reinforced/unanchored, time = 0, on_floor = TRUE, window_checks = TRUE), \
-	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/plasma/reinforced/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE) \
+	//austation begin -- glass floors
+	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/plasma/reinforced/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE), \
```
Comment (TheFakeElon) [inline code/game/objects/items/stacks/sheets/glass.dm:175]: Same here

File: code/game/objects/items/stacks/sheets/glass.dm (hunk 1 above)
Comment (jupyterkat) [reply code/game/objects/items/stacks/sheets/glass.dm:128]: Just added a "," for it to work

File: code/game/objects/items/stacks/sheets/glass.dm (hunk 2 above)
Comment (jupyterkat) [reply code/game/objects/items/stacks/sheets/glass.dm:175]: ","

File: code/game/objects/items/stacks/sheets/glass.dm (hunk 1 above)
Comment (TheFakeElon) [reply code/game/objects/items/stacks/sheets/glass.dm:128]: As in. What’s this about a full tile r-window.

Comment (jupyterkat) [reply code/game/objects/items/stacks/sheets/glass.dm:128]: It's the stack recipe for crafting the fulltile window when you click it inhand

Comment (jupyterkat) [reply code/game/objects/items/stacks/sheets/glass.dm:128]: I extended the comment to there because a comma is added

Comment (TheFakeElon) [reply code/game/objects/items/stacks/sheets/glass.dm:128]: Don’t we already have that recipe though :p

Comment (TheFakeElon) [reply code/game/objects/items/stacks/sheets/glass.dm:128]: Ah alright

**Pull request merged**